docs/extend/plugins_graphdriver.md

```diff
@@ -34,6 +36,16 @@ Plugins are not required to accept these options as the Docker Engine does not
 require that the plugin use this path or options, they are only being passed
 through from the user.
 
+A list of UID and GID mappings used for user namespaces is also provided. These
+are an array of mappings which are structured like so:
+```
```

Comment:
Can you add a `json` language hint to the fence here, and for other JSON examples?